src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main extends FigureSupplier {

```

Comment:
The Main class should not extend FigureSupplier. Instead, create an instance of FigureSupplier in the Main class to use its methods.